lib/src/screens/LaunchScreen.dart

```diff
@@ -0,0 +1,324 @@
+import 'package:flutter/material.dart';
+import 'package:gattai/src/screens/LogInFlow/Login_Email.dart';
+import 'package:gattai/src/screens/SignUpFlow/SignUp_Email_Screen.dart';
+import 'package:gattai/src/widgets/LogIn_Button.dart';
+import 'package:gattai/src/widgets/SignUp_Button.dart';
+import 'package:liquid_swipe/Constants/Helpers.dart';
+import 'package:liquid_swipe/liquid_swipe.dart';
+
+
+class MyApp extends StatelessWidget {
+
+  static const TextStyle greyStyle =
+  TextStyle(fontSize: 40.0, color: Colors.grey, fontFamily: "Product Sans");
+  static const TextStyle whiteStyle =
+  TextStyle(fontSize: 40.0, color: Colors.white, fontFamily: "Product Sans");
+
+  static const TextStyle boldStyle = TextStyle(
```

Comment:
Same here, I think that all these styles can be in a helper file with all the style objects to share them across the whole project.